apps/wallet-mobile/src/navigation.tsx

```diff
@@ -82,11 +81,11 @@ export const defaultStackNavigationOptions = (theme: Theme): StackNavigationOpti
 
 export const DEPRECATED_defaultStackNavigationOptions: StackNavigationOptions = {
   headerStyle: {
-    backgroundColor: COLORS.BACKGROUND_BLUE,
+    // backgroundColor: COLORS.BACKGROUND_BLUE,
     elevation: 0,
     shadowOpacity: 0,
   },
-  headerTintColor: '#fff',
+  headerTintColor: '#000000',
```

Comment:
it looks it shouldn't be used this one, it looks it can be migrated to the new defaultStack...